README.md

```diff
@@ -120,6 +128,18 @@ Pre-requisites
     ```
 
     </details>
+- Installing Android SDK emulator for Pixel 9.0
+    - ``` cd /Users/{{your user}}/Library/Android/sdk/cmdline-tools/latest/bin ```
+    - ``` ./sdkmanager --install "system-images;android-28;google_apis;x86" ```
+    - ``` echo "no" | ./avdmanager --verbose create avd --force --name "pixel_9.0" --device "pixel" --package "system-images;android-28;google_apis;x86" --tag "google_apis" --abi "x86" ```
```

Comment:
I would suggest creating a few different AVDs at different API levels so that people can experiment with different permission levels